fcm.gemspec

```diff
@@ -21,5 +21,5 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_runtime_dependency('httparty', '~> 0.10', '>= 0.10.0')
+  s.add_runtime_dependency('faraday','0.15.4')
```

Comment:
The dependency can be too strict for the apps that already using `faraday` in their `Gemfile`.